Helm Chart's ingress template broken after 0.3.2 updates

When Superset Helm Chart version 0.3.2 is applied with ingress enabled and configured it fails to install.

`values.yaml` ingress snippet used to install the superset Chart:

```
ingress:
  enabled: true
  annotations:
    kubernetes.io/ingress.class: nginx
  path: /
  pathType: ImplementationSpecific
  hosts:
    - superset.mydomain.com
  tls:
    - hosts:
        - superset.mydomain.com
      secretName: "certificate-mydomain"
```

### Expected results

```
> helm upgrade --install superset-test superset/superset -f values.yaml --version 0.3.2 --dry-run

### unrelated output removed ###

apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: superset-test
  labels:
    app: superset
    chart: superset-0.3.2
    release: superset-test
    heritage: Helm
  annotations:
    kubernetes.io/ingress.class: nginx
spec:
  tls:
    - hosts:
        - superset.mydomain.com
      secretName: certificate-mydomain
  rules:
    - host: superset.mydomain.com
      http:
        paths:
          - path: /
            pathType: ImplementationSpecific
            backend:
              service:
                name: superset-test
                port:
                  name: http
```

### Actual results
```
> helm upgrade --install superset-test superset/superset -f values.yaml --version 0.3.2 --dry-run
Release "superset-test" does not exist. Installing it now.
coalesce.go:163: warning: skipped value for extraEnv: Not a table.
Error: template: superset/templates/ingress.yaml:49:28: executing "superset/templates/ingress.yaml" at <.Values.ingress.path>: can't evaluate field Values in type interface {}
```

### Environment

- superset helm chart version: 0.3.2
- helm version: v3.5.4

### Checklist

- [*] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [*] I have reproduced the issue with at least the latest released version of superset.
- [*] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

Problem is caused by commit https://github.com/apache/superset/commit/a3a34773ae5c402bec5c6dbba0c9c453d518e147  to adapt the Ingress resource to the latest API version.

On line 49 of ingress.yaml, the `.Values.ingress.path` value is referenced while the context is set to `.Values.ingress.hosts` due to the `range` loop.

To use the root context while being in different context,use `$` as described in the Helm documentation
https://helm.sh/docs/chart_template_guide/variables/

--- a/helm/superset/Chart.yaml
+++ b/helm/superset/Chart.yaml
@@ -22,7 +22,7 @@ maintainers:
   - name: craig-rueda
     email: craig@craigrueda.com
     url: https://github.com/craig-rueda
-version: 0.3.2
+version: 0.3.3
 dependencies:
 - name: postgresql
   version: 10.2.0

--- a/helm/superset/templates/ingress.yaml
+++ b/helm/superset/templates/ingress.yaml
@@ -16,7 +16,6 @@
 #
 {{ if .Values.ingress.enabled -}}
 {{- $fullName := include "superset.fullname" . -}}
-{{- $ingressPath := .Values.ingress.path -}}
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
@@ -46,8 +45,8 @@ spec:
     - host: {{ . }}
       http:
         paths:
-          - path: {{ .Values.ingress.path }}
-            pathType: {{ .Values.ingress.pathType }}
+          - path: {{ $.Values.ingress.path }}
+            pathType: {{ $.Values.ingress.pathType }}
             backend:
               service:
                 name: {{ $fullName }}

